README.md

```diff
@@ -132,7 +135,7 @@ Running a **Full sync** using `-f` or `--full` option will not create any `custo
 ##### Download
 
    ```bash
-docker pull commercetools/commercetools-project-sync:3.2.1
+docker pull commercetools/commercetools-project-sync:3.2.2
```

Comment:
Should be 3.3.0? 🤔

Reply:
https://github.com/commercetools/commercetools-project-sync/pull/136/commits/9fe9f1540da19c32693c1725bedd2b671f5ffb47